PLANNED_FEATURES.md

```diff
@@ -1,10 +1,24 @@
 # Planned features
 
-This list contains rather high-prio features that will most likely be implemented in the near future.
+If you have ideas or wishes and don't find them on this list,
+feel free to start the discussion by raising a feature request.
 
-* Windows support
+## iceoryx v3.0 and later
+
+This is an incomplete list of possible features for later versions of
+iceoryx.
+
+* Windows command line support
+* Dynamic types
+* Sending dynamic types like vector, string, list
+* iceoryx tools to debug connections and communication content
+* user defined platform support
+* user defined memory provider to support hardware accelerators
 * Man page for RouDi and the runtime
-* Request-response communication
+* packages for linux distributions like debian, archlinux, gentoo
```

Comment:
If you have to change something else, please start every point with a capital letter (except the iceoryx tools... of course)